app/components/AnalystDashboard/AssessmentAssignmentTable.tsx

```diff
@@ -461,7 +473,7 @@ const AssessmentAssignmentTable: React.FC<Props> = ({ query }) => {
       <Box>
         <StyledText>
           Showing applications with status of “Received”, “Screening” and
```

Comment:
Just a nit, but we're missing a comma after "Screening" and should remove "and
```suggestion
          Showing applications with status of “Received”, “Screening”,
```